### PR TITLE
Blog post: My experience at the JS Crash Course Berlin.

### PR DESCRIPTION
**Pull request by mignonnesaurus**
Hello! :-)

This is a blog post about my Experience at the  [JS Crash Course Berlin](http://wtmberlin.com/javascript-crash-course/)

Thank you!

### CODE REVIEW

**Comment by Pisush** [comment]
Thanks for writing!
The post looks great, just a few styling edits and we're good to go 👍

File: _posts/2017-12-12-my-experience-at-js-crash-course.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: My experience at the JavaScript Crash Course Berlin
+date: 2017-12-12
+categories: story
+---
+
+I recently had the opportunity to attend the [JavaScript Crash Course Berlin](http://wtmberlin.com/javascript-crash-course/).
```
Comment (Pisush) [inline _posts/2017-12-12-my-experience-at-js-crash-course.markdown]: will yo be ok with renaming `JavaScript Crash Course Berlin` to `JavaScript Crash Course by Women Techmakers Berlin?`

File: _posts/2017-12-12-my-experience-at-js-crash-course.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+It was a 2 months course, the lectures were every Thursday from 7 PM to 10 PM and the goal of the course was
```
Comment (Pisush) [inline _posts/2017-12-12-my-experience-at-js-crash-course.markdown]: looks like you have some new lines slipped in the middle of this sentence and a few others :)

File: _posts/2017-12-12-my-experience-at-js-crash-course.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+categories: story
+---
+
+I recently had the opportunity to attend the [JavaScript Crash Course Berlin](http://wtmberlin.com/javascript-crash-course/).
+It was a 2 months course, the lectures were every Thursday from 7 PM to 10 PM and the goal of the course was 
+ _To introduce JavaScript and Node.js as a platform for backend applications._ 
+
+At first, when I signed up to the course, I was unsure if I was going to make it until the end. 8 weeks seemed like a 
+long time and December seemed so far away. How wrong I was! Time went by so quickly, and I felt so at peace during the
+course. Even when I had a rough Thursday, I always new everything would be better the moment I reached the
```
Comment (Pisush) [inline _posts/2017-12-12-my-experience-at-js-crash-course.markdown]: small typo: new -> knew

File: _posts/2017-12-12-my-experience-at-js-crash-course.markdown (hunk 1 above)
Comment (mignonnesaurus) [reply _posts/2017-12-12-my-experience-at-js-crash-course.markdown]: Yes, of course! :)

File: _posts/2017-12-12-my-experience-at-js-crash-course.markdown (hunk 2 above)
Comment (mignonnesaurus) [reply _posts/2017-12-12-my-experience-at-js-crash-course.markdown]: Yes, that's true! Fixed! 👍

**Comment by mignonnesaurus** [comment]
Thank you @Pisush! :tada:

**Pull request merged**